drivers/clock_control/Kconfig.stm32

```
@@ -164,7 +179,8 @@ config CLOCK_STM32_MCO1_SRC_PLLI2SCLK_DIV2
 
 config CLOCK_STM32_MCO1_SRC_SYSCLK
 	bool "SYSCLK"
-	depends on SOC_SERIES_STM32F1X || SOC_SERIES_STM32L4X
+	depends on SOC_SERIES_STM32F1X || SOC_SERIES_STM32L4X || \
+		   SOC_SERIES_STM32U5X
```

For consistency in the file:
```suggestion
	depends on SOC_SERIES_STM32F1X || \
	           SOC_SERIES_STM32L4X || \
		   SOC_SERIES_STM32U5X
```